import-service/src/catalogBatchProcess.js

```diff
@@ -0,0 +1,78 @@
+const { Client } = require("pg");
+const AWS = require("aws-sdk");
+
+const { PG_HOST, PG_PORT, PG_DATABASE, PG_USERNAME, PG_PASSWORD } = process.env;
+const dbOptions = {
+  host: PG_HOST,
+  port: PG_PORT,
+  database: PG_DATABASE,
+  user: PG_USERNAME,
+  password: PG_PASSWORD,
+  ssl: {
+    rejectUnauthorized: false, // to avoid warring in this example
+  },
+  connectionTimeoutMillis: 5000, // time in millisecond for termination of the database query
+};
+
+const catalogBatchProcess = async (event) => {
+  const client = new Client(dbOptions);
+  await client.connect();
+
+  const sns = new AWS.SNS({ region: "eu-west-1" });
+
+  try {
+    event.Records.forEach(async ({ body }) => {
+      console.log(body);
+      const { title, description, price, count } = JSON.parse(body);
+
+      const {
+        rows,
+      } = await client.query(
+        `insert into products (title, description, price) values ($1, $2, $3) returning id`,
+        [title, description, price]
+      );
+
+      const newId = rows[0].id;
+
+      await client.query(
+        `insert into stocks (product_id, count) values ($1, $2)`,
+        [newId, count]
+      );
+
+      await client.query(
```

Comment:
Not a bug. This query might be omitted by combining responses from two queries above.
As another option, you could you WITH statement from Postgres. So there might be only one request.